freqtrade/freqtradebot.py

```diff
@@ -1085,7 +1085,7 @@ def _notify_sell_cancel(self, trade: Trade, order_type: str) -> None:
             'open_rate': trade.open_rate,
             'current_rate': current_rate,
             'profit_amount': profit_trade,
-            'profit_percent': profit_percent,
+            'profit_ratio': profit_ratio,
```

Comment:
If we change the dict key here - we'll also have to change it in `webhook-config.md`

Reply:
see below